addons/mail/static/src/components/discuss_sidebar/discuss_sidebar.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.DiscussSidebar" owl="1">
         <div name="root" class="o_DiscussSidebar" t-attf-class="{{ className }}" t-ref="root">
             <div class="d-flex justify-content-center">
-                <button class="o_DiscussSidebar_startAMeetingButton btn btn-primary rounded" title="Start a meeting" t-on-click="discuss.onClickStartAMeetingButton" t-ref="startAMeetingButton">
+                <button class="o_DiscussSidebar_startAMeetingButton btn btn-primary rounded" title="Start a meeting" t-on-click="discuss.onClickStartAMeetingButton">
```

Comment:
make a separate PR to remove this one if it's now useless

Reply:
PR with all useless t-refs removed: https://github.com/odoo/odoo/pull/81801